pillar/elastalert.sls

```diff
@@ -475,6 +475,34 @@ elastic_stack:
                 filter:
                   - term:
                       fluentd_tag: edx.cms
+      - name: edx_s3_response_error
+        settings:
+          name: edX S3 Response Error
+          description: >-
+            An edX worker got an error from S3 while trying to export course
+            content to Git. This usually means that the uwsgi service needs to
```

Comment:
For edX the celery workers are managed with supervisor, not uwsgi. Also, this isn't exclusive to the workers, so a more accurate representation is that the S3 credentials have expired, so the process might need to be restarted or the credentials might need to be refreshed.

Reply:
OK, changed.